### DISCUSSION TITLE
Can we provide a better review assignment email subject.

### DISCUSSION THREAD

**Discussion by rjsparks** [ietf-tools/datatracker#3760] [Ideas]
See the thread starting at https://mailarchive.ietf.org/arch/msg/tools-discuss/MeFTTsnSBu4kaT3Xx5c9Jj_UXws.

There is a proposal, gaining support, to change the review assignment email subject line to
```
<team name> review assignment of <draft-foo> by <deadline>
```

**Comment by rjsparks** [comment]
It is possible to make these subject lines review team specific, but it would be better if all teams adopted the same format.

**Reply by evyncke** [reply]
Just looked at the code but it seems that the template is for the body and not for the Subject: line though

**Comment by russhousley** [comment]
This is totally fine by me.

**Reply by Spectre17** [reply]
Agreed. This seems like a helpful change. And the data is already there to make that happen.

**Comment by rjsparks** [comment]
Ted Lemon suggests also providing a calendar to subscribe to that shows assignments (by due date).

Might be hard to construct a meaningful time-of-day that would be normally calendar visible from any timezone.
Would "all-day, assignment date to due date" be useful?

**Comment by Abhayakara** [comment]
I’d like to suggest that the bit about it being a review assignment be in upper case so that it stands out. I realize that’s not aesthetically pleasing, but I’m afraid that this change without the caps may not be enough to make the subject line stand out.

**Reply by cjbc** [reply]
I personally prefer not to have the upper case, as I find that a bit aggressive (I don't receive that many e-mails with a subject in uppercase, and the few I get, are typically VERY important/urgent). The bottom line is that if all people start using uppercase to catch the attention of the receiver, we might end-up with all e-mail subjects being in uppercase ;)

In any case, this is just my opinion as individual.

**Reply by evyncke** [reply]
Same for me, I tend to filter out email messages that have an uppercase subject header as this is often spam

**Comment by evyncke** [comment]
I would go a little more personal though:

`Your <team name> review assignment of <draft-foo> by <deadline>`

or even

`<person name>'s  <team name> review assignment of <draft-foo> by <deadline>`

**Comment by kivinen** [comment]
I think the change is fine for the secdir, I myself never see those messages as I am secretary and datatracker do not send them to me (not even when I assign document for myself), so it is not directly affecting me. I don't like using UPPER CASE text as that tends to be marked as spam much easier (I think spamassasin gives you 0.5 points for having SUBJ_ALL_CAPS, but that would require all of subject to be caps).

Having calendar feed would be really good, and as this needs to be something that person authenticates and subscribes himself, providing ways of configuring it would be possible, thus adding timezone, or whether there should be alarm preset on it and how much earlier, to the subscription url should solve the issue of timezone and type of entry.

**Comment by evyncke** [comment]
I have some local code sending emails with Subject like 
`Subject: For Suzanne Woolf, Intdir Telechat review assigned: draft-ietf-bier-bar-ipa `

**Comment by cjbc** [comment]
Hi Éric,

That's great. Would it be possible to add the deadline (appending something
like "by XX-XX-XXXX") to the subject line?

Thanks!

Carlos

**Comment by evyncke** [comment]
See PR #4769
